src/main/java/org/zalando/nakadi/repository/db/EventTypeDbRepository.java

```diff
@@ -86,7 +92,7 @@ public EventType mapRow(final ResultSet rs, final int rowNum) throws SQLExceptio
     @Override
     public List<EventType> list() {
```

Comment:
and test https://github.com/zalando/nakadi/pull/386/commits/71d5574c65915a82f86caed7b2a2886c9b1d1516